### DISCUSSION TITLE
How to  increasing the number of triangles?

### DISCUSSION THREAD

**Discussion by d11122** [phoebe-project/phoebe2#514] [Questions]
Hi, when I run_compute the contact binary model, I got this warning, but I don't quite understand how this warning happened and how to change it?
“”SOLVER       WARNING received error from run_compute: Mesh failed or incomplete. Try increasing the number of triangles!.  lnprobability=-inf“”
Thanks you!

**Comment by kecnry** [comment]
You're looking for the `ntriangles` parameter in the compute options.  I will also push a change so that in the next major release, the error message will give the name of the parameter.  If increasing `ntriangles` doesn't seem to help, please send a minimal example to reproduce and we can try to track down if something else is going on that can be fixed.

**Reply by d11122** [reply]
thanks!
